### PR TITLE
Update ch99-01-04-01-voting-contract.md

### PR DESCRIPTION
**Pull request by OkoliEvans**
@enitrat  I made the changes you recommended here

### CODE REVIEW

File: src/ch99-01-04-01-voting-contract.md [hunk 1]
```diff
@@ -118,10 +119,20 @@ This process is identical for `account_1` and `account_2` in case you want to ha
 
 Before deploying, we need to declare the contract. We can do this with the `starkli declare` command:
 
+```bash
+starkli declare target/dev/starknetbook_chapter_2_Vote.sierra.json --rpc http://0.0.0.0:5050 --account ~/.starkli-wallets/deployer/account0_account.json --keystore ~/.starkli-wallets/deployer/account0_keystore.json
+```
+
+If your compiler version is lower than `compiler-version 2.2.0`, and if you encounter `compiler-version` error while using the command above, you can specify your compiler version in the command:
```
Comment (enitrat) [inline src/ch99-01-04-01-voting-contract.md:126]:
```suggestion
If the compiler version you're using is older than the one used by Starkli, which is synced and you encounter a `compiler-version` error while using the command above, you can specify a compiler version to use in the command by adding the `--compiler-version x.y.z` flag.

Comment (enitrat) [reply src/ch99-01-04-01-voting-contract.md:126]: and then you can remove the duplicated command below

File: src/ch99-01-04-01-voting-contract.md [hunk 2]
```diff
@@ -118,10 +119,20 @@ This process is identical for `account_1` and `account_2` in case you want to ha
...
+```
+
+If your compiler version is lower than `compiler-version 2.2.0`, and if you encounter `compiler-version` error while using the command above, you can specify your compiler version in the command: 
+
+
 ```bash
 starkli declare target/dev/starknetbook_chapter_2_Vote.sierra.json --compiler-version 2.0.1 --rpc http://0.0.0.0:5050 --account ~/.starkli-wallets/deployer/account0_account.json --keystore ~/.starkli-wallets/deployer/account0_keystore.json
 ```
 
+If you still encounter `compiler-version` error after specifying your compiler version, try upgrading Starkli using the command: `starkliup`
```
Comment (enitrat) [inline src/ch99-01-04-01-voting-contract.md:133]:
```suggestion
If you're still encountering issues with the compiler version, try upgrading Starkli using the command: `starkliup` to make sure you're using the latest version of starkli.
```

**Review by enitrat — COMMENTED** [review]
almost good 💪 just a few changes to make it lighter

File: src/ch99-01-04-01-voting-contract.md (hunk 2 above)
Comment (OkoliEvans) [reply src/ch99-01-04-01-voting-contract.md:133]: > 

I made the changes in this commit: https://github.com/cairo-book/cairo-book.github.io/pull/395

Comment (OkoliEvans) [reply src/ch99-01-04-01-voting-contract.md:133]: You really made the text better and more comprehensive

**Pull request closed without merging**